tests/testthat/test_featsel_selectFeaturesSequential.R

```diff
@@ -0,0 +1,18 @@
+context("selectFeaturesSequential")
+
+# This used to cause an error. Now ensure it does not.
+test_that("no crash with sffs", {
+  p <- mlbench::mlbench.waveform(1000)
+  dataset <- as.data.frame(p)
+  dataset  <- droplevels(subset(dataset, classes != 3))
+
+  mCT <- makeClassifTask(data = dataset, target = "classes")
+  ctrl = makeFeatSelControlSequential(method = "sffs", maxit = NA,alpha = 0.001)
+  mL <- makeLearner("classif.logreg", predict.type = "prob")
+  inner = makeResampleDesc("Holdout",stratify = TRUE)
+  lrn = makeFeatSelWrapper(mL, resampling = inner, control = ctrl)
+  outer = makeResampleDesc("CV", iters = 10,stratify = TRUE)
```

Comment:
Would that work with a smaller number of folds or a simple holdout? We try to keep the tests as quick as possible.